File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    answer = []
    
    zero = 0
    cnt = 0
    
    while len(s) != 1:
        case = ""
        zero_cnt = 0
        for i in s:
            if i == "1":
                case += "1"
            else:
                zero_cnt += 1
        zero += zero_cnt
        s = bin(len(s)-zero_cnt)[2:]
        cnt += 1
        
    answer = [cnt, zero]
    return answer

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    var answer = [];

    let zero = 0
    let cnt = 0
    
    while (s.length > 1) {
        let ch = ''
        let zeroCnt = 0
        for (let i = 0; i < s.length; i++) {
            if (s[i] == '1') {
                ch += '1'
            } else {
                zeroCnt += 1 
            }
        }
        zero += zeroCnt
        let len = (s.length - zeroCnt).toString(2)
        s = len
        cnt += 1
    }
    
    answer = [cnt, zero]
    return answer;
}